lib/app.js

```diff
@@ -173,9 +173,13 @@ if (navigator.mozSetMessageHandler) {
   navigator.mozSetMessageHandler('activity', function(activityRequest) {
```

Comment:
`navigator.setMessageHandler` doesnt exist, this is purely for activity support, if browsers dont support it it just doesnt run